java/src/jmri/jmrix/dccpp/swing/virtuallcd/VirtualLCDFrame.java

```diff
@@ -0,0 +1,81 @@
+package jmri.jmrix.dccpp.swing.virtuallcd;
+
+import java.util.ArrayList;
+import javax.swing.BoxLayout;
+import javax.swing.JTextField;
+
+import jmri.jmrix.dccpp.*;
+import jmri.util.JmriJFrame;
+
+/**
+ * Frame to image the DCC-EX command station's OLED display
+ *   Also sends request to DCC-EX to send copies of all LCD messages to this instance of JMRI
+ *
+ * @author Bob Jacobsen Copyright (C) 2023
+ */
+public class VirtualLCDFrame extends JmriJFrame implements DCCppListener  {
+
+    // private data
+    private DCCppTrafficController _tc = null;
+    private DCCppSystemConnectionMemo _memo;
+
+    public VirtualLCDFrame(DCCppSystemConnectionMemo memo) {
+        super();
+        _tc = memo.getDCCppTrafficController();
+        _memo = memo;
+        _tc.sendDCCppMessage(DCCppMessage.makeLCDRequestMsg(), null);        
+    }
+
+    /** 
+     * {@inheritDoc}
+     */
+    @Override
+    public void message(DCCppMessage msg) {
+    }
+    
+    /** 
+     * {@inheritDoc}
+     */
+    @Override
+    public void message(DCCppReply msg) {
+        if (msg.isLCDTextReply()) {
+            lines.get(msg.getLCDLineNumInt()).setText(msg.getLCDTextString());
+        }
+    }
+    
+    /** 
+     * {@inheritDoc}
+     */
+    @Override
+    public void notifyTimeout(DCCppMessage msg) {
+    }
+    
+    final static int TEXTFIELDLENGTH = 40;
+    final static int TOTALLINES = 8;
+    ArrayList<JTextField> lines;
+    
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    public void initComponents() {
+        super.initComponents();
+        getContentPane().setLayout(new BoxLayout(getContentPane(), BoxLayout.Y_AXIS));
+        
+        // initialize the list of display lines
+        lines = new ArrayList<>(9);
```

Comment:
Static analysis fails on:
`Error:  Low: VirtualLCDFrame.lines not initialized in constructor and dereferenced in jmri.jmrix.dccpp.swing.virtuallcd.VirtualLCDFrame.message(DCCppReply) [jmri.jmrix.dccpp.swing.virtuallcd.VirtualLCDFrame] At VirtualLCDFrame.java:[line 42] UWF_FIELD_NOT_INITIALIZED_IN_CONSTRUCTOR`

My suggestion is that you move this line to the constructor and make the field `lines` private final.